etc/icinga2/constants.conf.cmake

```diff
@@ -3,16 +3,30 @@
  * the other configuration files.
  */
 
-/* The directory which contains the plugins from the Monitoring Plugins project. */
+/*
+ * Array of directories which can contain plugins from the Monitoring Plugins
+ * project or other sources.
+ */
+const PluginPath = "@ICINGA2_PLUGINPATH@".split(":")
```

Comment:
We can't provide that as a default value. The split() needs to be moved, and the array must be rendered "as is".